Directory.Build.props

```diff
@@ -331,6 +331,7 @@
 
     <!-- Warnings -->
     <NoWarn>$(NoWarn);BCL0020</NoWarn> <!-- TODO: Remove BCL0020 once https://github.com/dotnet/arcade/pull/2158 is consumed. -->
```

Comment:
@stephentoub, this looks like it has likely been consumed by now.

Reply:
Thanks for the reminder.  I'll take care of it.